### ISSUE TITLE
Not registering Schema for model

### ISSUE THREAD

**Issue by gausie** [baugarten/node-restful#42]
I've got my models in folders, for example:

``` javascript
// Meeting model

var mongoose = require('mongoose')
  , restful = require('node-restful');

var meetingSchema = mongoose.Schema({
  date: Date,
  location: String,
  committee: mongoose.Schema.ObjectId
});

module.exports = restful.model('Meeting', meetingSchema).methods(['get','post']);
```

and then 

``` javascript
Meeting = require("./models/meeting.js");
Meeting.register(app, '/api/meetings');
```

However, for some reason I'm getting a `MissingSchemaError: Schema hasn't been registered for model "Committee".` error. Any ideas why?

**Comment by OmgImAlexis** [comment]
@baugarten can you close this as it was fixed in #49

**Comment by baugarten** [comment]
👍
